dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Test.scala

```diff
@@ -34,7 +34,7 @@ object Test {
   val imageSize = 224
 
   def main(args: Array[String]) {
-    testParser.parse(args, new TestParams()).map(param => {
+    testParser.parse(args, new TestParams()).foreach { param =>
```

Comment:
Generally the issue here is that map produces a new collection (and, in some cases is only lazily evaluated) while foreach will always execute and doesn't bother to create a pointless collection of Unit